### PR TITLE
Currency directive

### PR DESCRIPTION
**Pull request by nwhite89**
- ``<rb-currency>`` currently you can only use ``$`` 
- Will display 18291.19 as ``$18,291.19``
- Doesn't at cents when ``.00``

TP: https://rockabox.tpondemand.com/entity/8271

### CODE REVIEW

File: src/rb-date-time/demo/demo-state.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ define([
     function demoState ($stateProvider, $urlRouterProvider) {
         $stateProvider.state('rb-date-time', {
             url: '/rb-date-time',
-            controller: 'demo-ctrl as demoCtrl',
+            controller: 'demo-date-time-ctrl as demoCtrl',
```
Comment (chrisvxd) [inline src/rb-date-time/demo/demo-state.js]: typo

File: src/rb-date-time/demo/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ define([
             rbDateTime.name
         ])
         .config(demoState)
-        .controller('demo-ctrl', demoCtrl);
+        .controller('demo-date-time-ctrl', demoCtrl);
```
Comment (chrisvxd) [inline src/rb-date-time/demo/index.js]: typo

**Comment by chrisvxd** [comment]
I think the point of this directive is to take any numerical value, and show it as currency, with a configurable number of decimal places. This implementation requires us to define the decimal places in the controller each time. Here's how I would probably expect this to work:

`<rb-currency amount="1000" decimal-places="2" />` renders to `$1,000.00`
`<rb-currency amount="1000" decimal-places="0" />` renders to `$1,000`
`<rb-currency amount="1000.5122" decimal-places="2" />` renders to `$1,000.51`

Obviously the story says only show 2 decimal places, so that can be hard-coded

File: src/rb-date-time/demo/index.js (hunk 2 above)
Comment (nwhite89) [inline src/rb-date-time/demo/index.js]: no this is correct

Comment (chrisvxd) [inline src/rb-date-time/demo/index.js]: @nwhite89 this is the currency control, not datetime

Comment (chrisvxd) [inline src/rb-date-time/demo/index.js]: + it only needs to be unique in this module

Comment (chrisvxd) [inline src/rb-date-time/demo/index.js]: Oh wait, this is in the date-time component. Ignore this.

**Pull request closed without merging**